Rename rda folders

sbreitbart-NOAA opened the pull request:
<!---
Thanks for opening a PR. This commented text will **NOT** appear in the final PR. Toggle between Write and Preview to see what your PR will look like without the comments.
-->

# What is the feature?
*

# How have you implemented the solution?
*

# Does the PR impact any other area of the project, maybe another repo?
*

Schiano-NOAA commented:
Maybe we can talk about this in our meeting later, but what are your thoughts on separating this out to tables folder and figures folder. This could help scientists better organize their materials and when things can get extensive, find things easier. Figures can become a lot as well as tables (or table code), so I think there are many advantages to separating these out.

sbreitbart-NOAA commented:
> Maybe we can talk about this in our meeting later, but what are your thoughts on separating this out to tables folder and figures folder. This could help scientists better organize their materials and when things can get extensive, find things easier. Figures can become a lot as well as tables (or table code), so I think there are many advantages to separating these out.

New discussion/poll started to help answer this question: https://github.com/nmfs-ost/asar/discussions/281

sbreitbart-NOAA commented:
Note: based on user feedback, figures and tables will be split into separate folders.

Schiano-NOAA commented:
@sbreitbart-NOAA Is this setting up to split this into two separate tables and figures folders? I thought from the discussion we were going to separate them.

Schiano-NOAA reviewed (approved):
Looks good! Minor things to make changes (not impacting the function of the code). I added a few thoughts along the way too 😄

Schiano-NOAA commented on R/create_tables_doc.R:
```diff
@@ -1,517 +1,325 @@
+                "tbl-cap: !expr if(eval_", tab_shortname, ") ", tab_shortname, "_cap"
+              ),
+              glue::glue(
+                "include: !expr eval_", tab_shortname
```
Sorry just seeing this after my review. Yes cleanup-pkg is finished and things are fixed

Schiano-NOAA commented on R/create_figures_doc.R:
```diff
@@ -14,482 +16,151 @@
+create_figures_doc <- function(subdir = getwd(),
+                               figures_dir = getwd(),
+                               include_all = TRUE) {
+  if (!include_all) cli::cli_abort("Functionality for adding specific figures is still in development. Please set 'include_all' to true and edit the 09_figures.qmd file to remove specific figures from the report.")
```
Works for me 👍

sbreitbart-NOAA commented on inst/glossary/report_glossary.tex:
Just made this into [its own issue](https://github.com/nmfs-ost/asar/issues/286) and will address in a separate PR

sbreitbart-NOAA commented:
> Looks good! Minor things to make changes (not impacting the function of the code). I added a few thoughts along the way too 😄
> 
> Please rebase and let me know if you are getting all of these merge conflicts

Thanks Sam! I've addressed all but one, which I'll address in a future PR (https://github.com/nmfs-ost/asar/issues/286). Rebasing did take a while because I did get a lot of merge conflicts, but I sorted them out.
